omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/support/openmrs2_0/NameSupport2_0.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.support.openmrs2_0;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Vector;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.GlobalPropertyListener;
+import org.openmrs.api.context.Context;
+import org.openmrs.layout.LayoutSupport;
+import org.openmrs.module.webservices.rest.web.v1_0.template.openmrs2_0.NameTemplate2_0;
+import org.openmrs.serialization.SerializationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NameSupport2_0 extends LayoutSupport<NameTemplate2_0> implements GlobalPropertyListener {
+    private static NameSupport2_0 singleton;
+    private boolean initialized = false;
+    static Logger log = LoggerFactory.getLogger(NameSupport2_0.class);
+
+    private NameSupport2_0() {
+        if (Objects.isNull(singleton))
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

Reply:
okay, let me make the changes accordingly